solr/modules/jwt-auth/src/test/org/apache/solr/security/jwt/JWTAuthPluginTest.java

```diff
@@ -64,21 +64,30 @@
   // Shared with other tests
   static HashMap<String, Object> testJwk;
 
+  public static Path JWT_TEST_PATH() {
+    return getFile("solr/security").getParentFile().toPath();
+  }
+
   static {
-    // Generate an RSA key pair, which will be used for signing and verification of the JWT, wrapped in a JWK
+    // Generate an RSA key pair, which will be used for signing and verification of the JWT, wrapped
+    // in a JWK
     try {
       rsaJsonWebKey = RsaJwkGenerator.generateJwk(2048);
       rsaJsonWebKey.setKeyId("k1");
 
       testJwk = new HashMap<>();
       testJwk.put("kty", rsaJsonWebKey.getKeyType());
-      testJwk.put("e", BigEndianBigInteger.toBase64Url(rsaJsonWebKey.getRsaPublicKey().getPublicExponent()));
+      testJwk.put(
+          "e",
+          BigEndianBigInteger.toBase64Url(rsaJsonWebKey.getRsaPublicKey().getPublicExponent()));
```

Comment:
Some of this spotless formatting is so awkward, maybe we could go with a slightly longer line length to make this less bad? Probably not important right now anyway, but thinking ahead to when we apply the plugin other places.

Reply:
Can we please apply formatting in a separate PR/commit and then exclude this from `.git-blame-ignore-revs` ?

Reply:
Did a shot in #611 
I first applied Spotless on JWT classes while still in solr-core. We can merge that one commit as a separate PR first.
Then rebased this branch on top, with some quashing etc to make the edit history more clean.
Hope you like it :) 

I did not dare to force-push the same to this same branch. No idea what would happen to the PR and review comment history if I did. So we can close this and finalize in #611